Makefile

```diff
@@ -70,23 +70,25 @@ generate-api:
 # is mounted under /home/notebook-user/local/ when the image is started with
 # docker-start-api or docker-start-jupyter
 
+DOCKER_IMAGE ?= unstructured-api:dev
+
 .PHONY: docker-build
 docker-build:
-	PIP_VERSION=${PIP_VERSION} PIPELINE_FAMILY=${PIPELINE_FAMILY} PIPELINE_PACKAGE=${PIPELINE_PACKAGE} ./scripts/docker-build.sh
+	PIP_VERSION=${PIP_VERSION} PIPELINE_PACKAGE=${PIPELINE_PACKAGE} DOCKER_IMAGE_NAME=${DOCKER_IMAGE} ./scripts/docker-build.sh
 
 .PHONY: docker-start-api
 docker-start-api:
-	docker run -p 8000:8000 --mount type=bind,source=$(realpath .),target=/home/notebook-user/local -t --rm pipeline-family-${PIPELINE_FAMILY}-dev:latest --log-config logger_config.yaml --host 0.0.0.0 --port 8000
+	docker run -p 8000:8000 --mount type=bind,source=$(realpath .),target=/home/notebook-user/local -t --rm ${DOCKER_IMAGE} --log-config logger_config.yaml --host 0.0.0.0 --port 8000
```

Comment:
other nit: are these `make` targets used in the current CI process? if not, i'd lean towards leaving as is for consistency with other pipeline- repos.

more context: these make targets had been more oriented towards local dev in the past, and are basically templated outputs from the pipeline- standard (something created from https://github.com/Unstructured-IO/pipeline-template/ ).

that said, it's not a huge deal to stray from that -- bulk updates across pipeline-api's are still possible with the right scripting, just want to make sure.

Reply:
nope, not necessary for ci! was in here and doing some clean up,  also wasn't sure if we wanted to keep calling this image pipeline-general vs unstructured-api

I do think being able to specify/override the docker image build name by variable is pretty valuable, but to your point maybe a change we do across all of them.

Reply:
yeah, that's a good point, and probably worth generalizing this across the other pipeline- repos. just need to remember to do that :)

Reply:
cool! will add that to a the ticket I'm writing for the platform variable and we can knock both out at once.